diff --git a/src/PhpWord/Writer/Word2007/Element/Footnote.php b/src/PhpWord/Writer/Word2007/Element/Footnote.php
index b59fc5e1cc..77073a239d 100644
--- a/src/PhpWord/Writer/Word2007/Element/Footnote.php
+++ b/src/PhpWord/Writer/Word2007/Element/Footnote.php
@@ -51,7 +51,7 @@ public function write(): void
         $xmlWriter->endElement(); // w:rStyle
         $xmlWriter->endElement(); // w:rPr
         $xmlWriter->startElement("w:{$this->referenceType}");
-        $xmlWriter->writeAttribute('w:id', $element->getRelationId());
+        $xmlWriter->writeAttribute('w:id', $element->getRelationId() + 1);
         $xmlWriter->endElement(); // w:$referenceType
         $xmlWriter->endElement(); // w:r
 
diff --git a/src/PhpWord/Writer/Word2007/Part/Footnotes.php b/src/PhpWord/Writer/Word2007/Part/Footnotes.php
index 9624ab7459..e284c674b5 100644
--- a/src/PhpWord/Writer/Word2007/Part/Footnotes.php
+++ b/src/PhpWord/Writer/Word2007/Part/Footnotes.php
@@ -141,7 +141,7 @@ public function setElements($elements)
     protected function writeNote(XMLWriter $xmlWriter, $element): void
     {
         $xmlWriter->startElement($this->elementNode);
-        $xmlWriter->writeAttribute('w:id', $element->getRelationId());
+        $xmlWriter->writeAttribute('w:id', $element->getRelationId() + 1);
         $xmlWriter->startElement('w:p');
 
         // Paragraph style